systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java

```diff
@@ -244,6 +245,66 @@ private void testDockerImagesForKafkaConnect() {
         LOGGER.info("Docker images verified");
     }
 
+    @Test
+    void testSecretsWithKafkaConnectWithTlsAuthentication() throws Exception {
+        final String userName = "user-example";
+
+        testMethodResources().tlsUser(KAFKA_CLUSTER_NAME, userName).done();
+
+        StUtils.waitForSecretReady(userName);
+
+        Secret userSecret = kubeClient().getSecret("user-example");
+        Secret clusterSecret = kubeClient().getSecret("connect-tests-cluster-ca-cert");
```

Comment:
Out of curiosity, you use the name of the secrets to get `userSecret` and `clusterSecret`. But the only thing you use them for is `.getMetadata().getName()` -> i.e. to get the name again. Why not just use the name directly?

Reply:
make sense